utils/query-string.js

```diff
@@ -34,6 +34,17 @@ function queryparser(query) {
 
   regex.lastIndex = 0;
   return params;
-}
+};
 
-module.exports = queryparser;
+exports.stringify = function stringify(obj) {
+  let string = '';
+  for (const prop in obj) {
+    if (string) {
+      string += '&';
+    }
```

Comment:
Minor: Instead of checking this every iteration, just truncate the final `'&'` char afterwards.

Reply:
Done.